source/D0323R11.bs

```diff
@@ -1635,7 +1636,7 @@ object of type `T` with the expression `std::forward<U>(v)`.
 *Throws*: Any exception thrown by the selected constructor of `T`.
 
 *Remarks*: `is_trivially_constructible_v<T, U>` is `true`,
-this constructor is trivial.
+this constructor is trivial. **[XXX specify constexpr, not trivial]**
```

Comment:
There's a few of these.

Reply:
Yeah, we need to decide how to word it in LWG. I didn't have time to get them right, so left reminders.

Reply:
A constructor like `expected(U&&) can't be trivial: it has to be user-defined, which precludes trivial.